src/xcode/ENA/ENA/Source/Scenes/Checkin/CheckInDetail/TraceLocationDetailViewModel.swift

```diff
@@ -25,7 +25,13 @@ final class TraceLocationDetailViewModel {
 		self.locationAddress = traceLocation.address
 		self.locationDescription = traceLocation.description
 		self.shouldSaveToContactJournal = store.shouldAddCheckinToContactDiaryByDefault
-		self.selectedDurationInMinutes = traceLocation.suggestedCheckoutLength
+		// max duration in the picker is 23:45
+		let maxDurationInMinutes = (23 * 60) + 45
```

Comment:
I like that you displayed it like this, so it is not a magic number but easy to understand 👍